import_export_celery/models/exportjob.py

```diff
@@ -110,6 +110,9 @@ def get_queryset(self):
         pks = json.loads(self.queryset)
         return self.get_content_type().model_class().objects.filter(pk__in=pks)
 
+    def get_queryset2(self,pks):
+        return self.get_content_type().model_class().objects.filter(pk__in=pks)
+
```

Comment:
Never copy code. Literally, NEVER do it. You've copied and pasted from the line above.

Find a way to do this without code duplication and with better names.

Reply:
I am not using "get_queryset", so we can replace it with "get_queryset2"